cves/kernel/CVE-2013-0290.yml

```diff
@@ -184,8 +190,11 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    TCP specification violation, vulnerability in the socket buffers being 
+    passed into the system. When the SKB is of zero-length it should be skipped
+    but the missing check results in a DOS.
+  answer: true
```

Comment:
Did you come to this conclusion after researching the CVE or was this mentioned in a bug report or commit?